README.md

```diff
@@ -121,3 +121,29 @@ Transformers with a * support the arg `unique` which will append an incrementing
     }
   },
 ```
+
+Transformers with a † support deterministic generation by setting `deterministic: true` and providing an `id-column` argument. This ensures the same input and ID always generate the same fake data.
+
+Example of deterministic name generation:
+```json
+{
+  "data_category": "Pii",
+  "description": "user's first name",
+  "name": "first_name",
+  "transformer": {
+    "name": "FakeFirstName",
+    "args": {
+      "deterministic": "true",
+      "id-column": "user_account_id"
```

Comment:
can `id-column` be `id_column` to be consistent with other properties like `data_category`?